test/ui-2/test/cropui.test.js

```diff
@@ -0,0 +1,46 @@
+const timeout = 300000 ;
+const fs = require('fs');
+beforeAll(async () => {
+  path = fs.realpathSync('file://../examples/index.html');
+  await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
+  await page.setViewport({ width: 1500, height: 700});
+});
+describe('Crop UI', ()=>{
+  test('Crop UI', async()=>{
+    //Wait for the load step to get loaded
```

Comment:
Please add a space at the start of every comment. Also, add a full stop at the end of sentences. We want the comments to be professional and readable.